### PR TITLE
Add support for Diff mode. Show difference

### PR DESCRIPTION
**Pull request by mastervolkov**
Can see changes in diff mode (-D)

### CODE REVIEW

**Comment by tpo** [comment]
Awesome!!!! Tested, works -> +1 !
Thanks a lot for this @mastervolkov

**Comment by particledecay** [comment]
The code looks good. I'll test this out and see if I can merge it in tonight. Thanks a bunch @mastervolkov!

File: json_patch.py [hunk 1]
```diff
@@ -248,6 +248,16 @@ def run(self):
...
+            dump_kwargs = {}
+            if self.pretty_print:
+                dump_kwargs.update({'indent': 4, 'separators': (',', ': ')})
+
+            result['diff'] = dict(
+                    before=self.json_doc,
+                    after=json.dumps(self.patcher.obj, **dump_kwargs),
+                    before_header='%s (content)' % self.module.params['src'],
+                    after_header='%s (content)' % self.module.params['src'],
+                )
```
Comment (particledecay) [inline json_patch.py:260]: Nitpick: please unindent lines 256-260

**Comment by mastervolkov** [comment]
Lines unindented

File: json_patch.py (hunk 1 above)
Comment (mastervolkov) [reply json_patch.py]: Ready

**Pull request merged**